src/encoded/static/components/search.js

```diff
@@ -1169,9 +1169,19 @@ export class ResultTable extends React.Component {
     constructor(props) {
         super(props);
 
+        // Make an array of all assemblies found in all files in the search results.
+        let assemblies = [];
+        const results = this.props.context['@graph'];
```

Comment:
Can also just use `props.context['@graph']` since `props` is passed in as a parameter, though I know we use `this.props` for this situation all over the place, and that does work fine.